dist/wysihtml5-0.4.1pre.js

```diff
@@ -8602,10 +8602,10 @@ wysihtml5.views.View = Base.extend(
       }, 0);
     });
 
-    // --------- neword event ---------
+    // --------- newword event ---------
     dom.observe(element, "keyup", function(event) {
       var keyCode = event.keyCode;
-      if (keyCode === wysihtml5.SPACE_KEY || keyCode === wysihtml5.ENTER_KEY) {
+      if (keyCode === wysihtml5.SPACE_KEY || (!isChrome && keyCode === wysihtml5.ENTER_KEY)) {
         that.parent.fire("newword:composer");
```

Comment:
Something about the `newword:composer` event firing in chrome is causing the bug in question. I tried digging in deeper, but couldn't pinpoint the issue at its core. Seems like adding a check here is good enough to alleviate the problem, and still preserves the both the Enter key and autolinking capabilities of the editor.